libfulltext/response.py

```diff
@@ -26,12 +45,22 @@ def save_to_file(response, filename):
         for chunk in response.iter_content(chunk_size=128):
             file.write(chunk)
 
+
 def verify(response, expected_content_type):
-    """Verify a response (currently only the status code and content type header)"""
+    """Verify a response (currently only the status code and content type header)
+
+    Args:
+        response:               requests.Response object from an API request
+        expected_content_type:  expected Content-Type as string (e.g. 'application/pdf')
+
+    Raises:
+        requests.exceptions.InvalidHeader: if content type undeclared or unexpected
+    """
     response.raise_for_status()
 
     if 'Content-Type' not in response.headers or \
             response.headers['Content-Type'] != expected_content_type:
-        raise requests.exceptions.InvalidHeader('Content-Type is not application/pdf, '
-                                                'possibly because you do not have access to '
-                                                'this document.')
+        raise requests.exceptions.InvalidHeader('Content-Type is not ' +
+                                                expected_content_type + ', '
```

Comment:
No need for either `+`. Python concatenates strings inside brackets.

Reply:
Hm, nose and flake complain about `E999 SyntaxError: invalid syntax` upon removal of the second `+`

Reply:
Oh, I'm sorry I misread `expected_content_type` by `'expected_content_type'` for some reason.

Reply:
Should be all good in the way it was.